conf/messages.cy

```diff
@@ -0,0 +1,153 @@
+# Site-wide Messages
+# ----------------------------------------------------------
+service.name = Coronavirus Job Support Scheme calculator
+
+site.back = Back
+site.delete = Delete
+site.hidden-delete = Delete {0}
+site.hidden-edit = Change {0}
+site.no = No
+site.yes = Yes
+site.continue = Continue
+site.tryAgain = Try again
+site.startAgain = Start again
+site.confirm = Confirm
+site.saveAndContinue = Save and continue
+site.submitAndContinue = Submit and continue
+site.textarea.char_limit = (Limit is {0} characters)
+site.govuk = GOV.UK
+site.signOut = Sign out
+site.getHelpWithThisPage = Get help with this page
+site.notProvided = Not Provided
+site.incomplete = Incomplete
+site.change = Change
+site.confirmAndSend = Confirm and send
+site.acceptAndSend = Accept and send
+
+footer.accessibility   = Accessibility statement
+footer.cookies         = Cookies
+footer.privacy         = Privacy policy
+footer.termsConditions = Terms and conditions
+footer.govukHelp       = Help using GOV.UK
+
+language.cymraeg = Cymraeg
+language.change.cymraeg = Newid yr iaith i'r Gymraeg
+language.english = English
+language.change.english = Change the language to English
+language.switcher = Language switcher
+
+# Timeout Messages
+# ----------------------------------------------------------
+timeout.title = You’re about to be signed out
+timeout.message = For security reasons, you will be signed out of this service in
+timeout.keepAlive = Stay signed in
+timeout.signOut = Sign out
```

Comment:
we're not implementing timeout in the service are we? if not, then we don't need this